### DISCUSSION TITLE
Clone Frame moves around in POE2

### DISCUSSION THREAD

**Discussion by HB1man81** [Lailloken/Exile-UI#495] [General]
Hi, Love the UI, using it for POE2

I am interested in the Clone Frames, putting buffs on top of my characters health and mana

However when I do this the clone 'appears' to move around in game... the reason for this is that the character does not appear to be always right in the middle of the screen when moving around

Is this how it is in POE1? any ways around it?

**Comment by Lailloken** [comment]
Yeah, it's the same in PoE1, and it's due to character movement along the z-axis because of terrain or certain slow-effects like quicksand. The only way around it is to place the clone-frame to the left/right of the hp and mana bars, or maybe underneath the character.

**Reply by HB1man81** [reply]
Ok thanks for quick reply.
